thirdeye/thirdeye-frontend/app/pods/components/detection-health/template.hbs

```diff
@@ -0,0 +1,84 @@
+<ul class="te-horizontal-cards__single-card">
+  <li class="te-horizontal-cards__card-title">
+    <label for="select-dimension" class="control-label te-label--no-bottom">
+      30-day status
+      <span>
+        <i class="glyphicon glyphicon-question-sign"></i>
+        {{#tooltip-on-element class="te-tooltip"}}
+          The overall health status of your detection configuration is based on
+          the most recent 30 days.
+        {{/tooltip-on-element}}
+      </span>
+    </label>
+  </li>
+  <li class="te-horizontal-cards__card-number{{overall.label}}">
+    {{overall.status}}
+  </li>
+  <a role="button" tabindex="0" class=".te-button--link">
+    view details
+    {{#bs-popover placement="bottom" title="Detection health"}}
+      <p>Overall health of the metrics that monitor detection performance.</p>
+
+      <div class="te-horizontal-metrics__container">
+        <ul class="te-horizontal-metrics__metric">
+          <li class="te-horizontal-metrics__number">{{executionTime}}</li>
+          <li class="te-horizontal-metrics__title">Execution Time</li>
```

Comment:
how about calling it "health report time" or something like that? IMO, calling it execution time might be confused with the "last detection execution time".

Reply:
we can keep it for now and then see what can be improved.